data/gegno/gegno missions.txt

```diff
@@ -0,0 +1,103 @@
+
+mission "First Contact: Gegno"
+	landing
+	source "Giaru Gegno"
+	to offer
+		has "First Contact: Quarg: offered"
+	on offer
+		log "Factions" "Gegno" `The Gegno are a humanoid species to the galactic east of the core, split amongst themselves into branches. One branch is the Gegno Vi, who are spoken of to be strong and forceful brutes. The other is the Gegno Scin, mentioned to be progressers of research and technology. Apart from them is a complacent civilian population.`
+		conversation
+			"After jumping through several systems of hostile alien ships, a Quarg ring world is a sight of relief to you. From the looks of it, this ringworld is much newer than that of the one in human space, on top of being completed. Millions of Quarg probably inhabit this ring, but what's more interesting to your eye is that some of the aliens also are present. They are also humanoid, and the most common figures you come across are either slightly above or below an average human's height. Their skin is a smooth, light gray, adorned with different sorts of leather that remind you of medieval-style human clothing."
```

Comment:
This actually reminds me- part of the idea is that the gegno occupy planets and territory of a much older long gone race- will have to find a way to fit that into the dialog if necessary.

Reply:
Some planet dialogs hint to this as well, but won't be accessible until later on. Perhaps I should add the idea of such an old ringworld being questionable for. anew species to the player

Reply:
If it's the case that it's an older ring being repurposed for the Gegno-watching Quarg, then would just need to remove the mentions of the ring being newer than the one in human space (I don't think the player character could judge that to be the case anyway, since I don't imagine ringworlds show many signs of the passage of time if there are Quarg living in them keeping them pristine as one would expect).
Other than that, the population mentions should be scaled up since this is a full ring, and there'd be a lot of Quarg in here (if it's an older ring, would for sure have their trillions pop, as Alta Hai alludes).